NitroxModel/Discovery/InstallationFinders/EpicGamesInstallationFinder.cs

```diff
@@ -1,37 +1,51 @@
-﻿using System;
+using NitroxModel.Discovery.Models;
+using System;
 using System.Collections.Generic;
 using System.IO;
 using System.Text.RegularExpressions;
 
-namespace NitroxModel.Discovery.InstallationFinders
+namespace NitroxModel.Discovery.InstallationFinders;
+
+public class EpicGamesInstallationFinder : IGameFinder
 {
-    public class EpicGamesInstallationFinder : IFindGameInstallation
+    private static readonly Regex installLocationRegex = new("\"InstallLocation\"[^\"]*\"(.*)\"", RegexOptions.Compiled);
```

Comment:
`RegexOptions.Compiled` is only really advised for Regex expressions which are used thousands of times. See [here](https://learn.microsoft.com/en-us/dotnet/standard/base-types/best-practices#interpreted-vs-compiled-regular-expressions)